src/community/monitor-micrometer/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.geoserver</groupId>
+    <artifactId>community</artifactId>
+    <version>2.27-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.geoserver.community</groupId>
+  <artifactId>gs-monitor-micrometer</artifactId>
+  <packaging>jar</packaging>
+  <name>Micrometer Monitor Extension</name>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.micrometer</groupId>
+        <artifactId>micrometer-bom</artifactId>
+        <version>1.14.4</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.geoserver.extension</groupId>
+      <artifactId>gs-monitor-core</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
```

Comment:
since you stated in the docs the monitor extension must be installed; I think this should be a `runtime` scope (in which case the scope can be omitted, that being the default)

Reply:
Thanks for the suggestion! 
Since `compile` is the default scope in Maven, I can simply omit it. 
However, I cannot use `runtime` because the code has direct imports from `gs-monitor-core`, which are needed at compile-time. 
Updated the PR accordingly.